modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -2293,6 +2298,14 @@ private void _pingTrackbacks(
 		}
 	}
 
+	private String _sanitizeUrlTitle(String urlTitle) {
+		while (urlTitle.startsWith(StringPool.SLASH)) {
+			urlTitle = urlTitle.substring(1);
+		}
+
+		return urlTitle;
+	}
+
```

Comment:
Are the tests passing? You're checking that things like `/////foo` are sanitized to `foo`, but this piece of code is no longer doing that. I'd expect this to be something like:
```
int i = 0;

while (i < urlTitle.length() && urlTitle.charAt(i) == SLASH) {
    i++;
}

return urlTitle.substring(i);
```

Reply:
yes, tests are passing

Reply:
I've double check the tests:

`
com.liferay.blogs.service.test.BlogsEntryLocalServiceTest > testAddEntryWithURLTitleWithSlashPrefix STARTED
com.liferay.blogs.service.test.BlogsEntryLocalServiceTest > testAddEntryWithURLTitleWithSlashPrefix PASSED
...
com.liferay.blogs.service.test.BlogsEntryLocalServiceTest > testUpdateEntryWithURLTitleWithSlashPrefix STARTED
com.liferay.blogs.service.test.BlogsEntryLocalServiceTest > testUpdateEntryWithURLTitleWithSlashPrefix PASSED

70 tests completed, 1 failed
There were failing tests. See the report at: file:///Users/roberto/work/open/liferay-portal/modules/apps/blogs/blogs-test/build/reports/tests/testIntegration/index.html
....
BUILD SUCCESSFUL in 4m 36s
479 actionable tasks: 96 executed, 1 from cache, 382 up-to-date
➜  ~/work/open/liferay-portal/modules/apps/blogs/blogs-test git:(LPD-25224) date
jueves,  9 de mayo de 2024, 15:41:40 CEST`

Reply:
Ohhh, I had read the while loop as if it was an `if` :facepalm: Sorry man.